[2023-10-02T03:42:20.580+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_Pipeline.clean_dataframe scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2023-10-02T03:42:20.620+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_Pipeline.clean_dataframe scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2023-10-02T03:42:20.622+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T03:42:20.624+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-10-02T03:42:20.626+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T03:42:20.678+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): clean_dataframe> on 2023-10-01 00:00:00+00:00
[2023-10-02T03:42:20.705+0000] {standard_task_runner.py:52} INFO - Started process 205 to run task
[2023-10-02T03:42:20.737+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Valdi_Pipeline', 'clean_dataframe', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Rivaldi_Valensia_DAG.py', '--cfg-path', '/tmp/tmplltmed0i', '--error-file', '/tmp/tmp_swtjop4']
[2023-10-02T03:42:20.747+0000] {standard_task_runner.py:80} INFO - Job 14: Subtask clean_dataframe
[2023-10-02T03:42:21.257+0000] {task_command.py:371} INFO - Running <TaskInstance: Valdi_Pipeline.clean_dataframe scheduled__2023-10-01T00:00:00+00:00 [running]> on host 5f5d0b98abe4
[2023-10-02T03:42:21.846+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Valdi
AIRFLOW_CTX_DAG_ID=Valdi_Pipeline
AIRFLOW_CTX_TASK_ID=clean_dataframe
AIRFLOW_CTX_EXECUTION_DATE=2023-10-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-01T00:00:00+00:00
[2023-10-02T03:42:22.945+0000] {logging_mixin.py:115} INFO - -------Data Saved------
[2023-10-02T03:42:22.949+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-10-02T03:42:23.041+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Valdi_Pipeline, task_id=clean_dataframe, execution_date=20231001T000000, start_date=20231002T034220, end_date=20231002T034223
[2023-10-02T03:42:23.121+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-02T03:42:23.279+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
